form/src/main/java/com/ritense/form/web/rest/impl/FormIoFormManagementResource.java

```diff
@@ -55,6 +55,11 @@ public ResponseEntity<? extends FormDefinition> getFormDefinitionById(@PathVaria
             .orElse(ResponseEntity.noContent().build());
     }
 
+    @Override
+    public ResponseEntity<? extends Boolean> getExistsByName(@PathVariable String name) {
+        return ResponseEntity.ok(formDefinitionService.getFormDefinitionByName(name).isPresent());
```

Comment:
<? extends ...> really needed?

Reply:
Yes I have made a FE validator that calls this endpoint to see if the name is available or not. Alternatively this check is skipped and the output of the create call is used to check if the name was available, but that would require some error message checking to make sure I report the right reason why it wasn't created. Also the benefit of this is that the initial value of the form can be checked when the modal opens, rather than when the submit button is pressed.